requirements.txt

```diff
@@ -1,2 +1,2 @@
-qumulo_api~=4.1
+qumulo_api~=5.1
```

Comment:
If we do maintain backward compatibility, we probably don't need this any more

Reply:
We could just drop the version pinning entirely? I think that'd work; our internal tests will catch future API endpoint breakage.

Reply:
Yeah, I think you are right